src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/PartiallyDeterminedHaplotypeComputationEngine.java

```diff
@@ -249,21 +188,12 @@ We iterate over every ref position and select single alleles (including ref) fro
             }
         }
 
-        if (outputHaplotypes.size() > MAX_PD_HAPS_TO_GENERATE) {
```

Comment:
this check got removed but i think it was still necessary to catch the 
`if (!pileupArgs.determinePDHaps) { ` path above

Reply:
ps. it only was in the if statement above because i found in testing the second path could end up taking hours and hours to process combinatorial numbers of haplotypes so you need to be able to cut it off before it explodes...

Reply:
fixed/restored